service/history/timerQueueProcessor.go

```diff
@@ -209,6 +212,15 @@ func (t *timerQueueProcessorImpl) NotifyNewTimer(timerTasks []persistence.Task)
 			t.minPendingTimer = ts
 			updatedMinTimer = true
 		}
+
+		switch task.GetType() {
+		case persistence.TaskTypeDecisionTimeout:
+			t.metricsClient.IncCounter(metrics.TimerTaskDecisionTimeoutScope, metrics.NewCounter)
```

Comment:
Debatable. these are number of newly created timers for each of those timeouts(DT/AT/UT). So the scope has newly created, processed count. Also i was aggregating them at timer-q-processor scope as well.